### PR TITLE
nightly build of main and develop

### PR DESCRIPTION
**Pull request by dvviktordelev**
Closes https://github.com/datavisyn/github-maintenance/issues/55

### Developer Checklist (Definition of Done)

**Issue**

- [ ] All acceptance criteria from the issue are met
- [ ] Tested in latest Chrome/Firefox

**UI/UX/Vis**

- [ ] Requires UI/UX/Vis review
  - [ ] Reviewer(s) are notified (_tag assignees_)
  - [ ] Review has occurred (_link to notes_)
  - [ ] Feedback is included in this PR
  - [ ] Reviewer(s) approve of concept and design

**Code**

- [ ] Branch is up-to-date with the branch to be merged with, i.e., develop
- [ ] Code is cleaned up and formatted
- [ ] Unit tests are written (frontend/backend if applicable)
- [ ] Integration tests are written (if applicable)

**PR**

- [ ] Descriptive title for this pull request is provided (will be used for release notes later)
- [ ] Reviewer and assignees are defined
- [ ] Add type label (e.g., *bug*, *feature*) to this pull request
- [ ] Add release label (e.g., `release: minor`) to this PR following [semver](https://semver.org/)
- [ ] The PR is connected to the corresponding issue (via `Closes #...`)
- [ ] [Summary of changes](#summary-of-changes) is written

### Summary of changes

- 

### Additional notes for the reviewer(s)

-  
Thanks for creating this pull request 🤗

### CODE REVIEW

File: .github/workflows/build_nightly.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+on:
+  workflow_dispatch:
+    inputs:
+      cypress:
+        description: 'Run cypress'
+        required: false
+        type: boolean
+        default: false
+  push:
```
Comment (puehringer) [inline .github/workflows/build_nightly.yml:11]: Do we really need to call it on push?

File: .github/workflows/build_nightly.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  schedule:
+    - cron: '15 1 * * *'
+
+jobs:
+  build-main:
+    uses: datavisyn/github-workflows/.github/workflows/build-node-python.yml@new_deployment
+    secrets: inherit
+    with:
+      cypress_enable: true
+      cypress_run_because_flag: ${{ inputs.cypress == true }}
```
Comment (puehringer) [inline .github/workflows/build_nightly.yml:21]: The nightly builds should always do the full cypress stuff (they probably do so automatically due to the branch names)

File: .github/workflows/build_nightly.yml (hunk 1 above)
Comment (dvviktordelev) [reply .github/workflows/build_nightly.yml:11]: Will remove it.

File: .github/workflows/build_nightly.yml (hunk 2 above)
Comment (dvviktordelev) [reply .github/workflows/build_nightly.yml:21]: Meaning setting cypress_run_because_flag: true ?

**Review by puehringer — DISMISSED** [review]

File: .github/workflows/build_nightly.yml (hunk 2 above)
Comment (puehringer) [reply .github/workflows/build_nightly.yml]: I just set the cypress flags to true, i.e. the nightly builds should always run cypress. For the rest, we have the normal build.yml.

**Pull request merged**